tiledb/sm/array/array.h

```diff
@@ -520,6 +520,9 @@ class Array {
   /** Load array directory for non-remote arrays */
   ArrayDirectory& load_array_directory();
 
+  /** Return the max tile size for the array. */
```

Comment:
Let's make max_tile_sizes a return value and not an input parameter.

Reply:
method name/prototype and usage changed to return single value currently being consumed by only client.